app/helpers/session_timeout_warning_helper.rb

```diff
@@ -18,35 +18,6 @@ def timeout_refresh_path
     )&.html_safe # rubocop:disable Rails/OutputSafety
   end
 
-  def auto_session_timeout_js
```

Comment:
I just inlined to the `layouts/_base.html.erb`, there didn't seem a good reason to keep them as helper methods to me